### PR TITLE
chore: add convenient script for checking node state via /health endpoint

### PR DESCRIPTION
**Pull request by NagyZoltanPeter**
Copied and enhanced the script chkhealth.sh added to nwaku.
It supposed to easy the way one can check node health state... hope to lower our node-help channel requests...

### CODE REVIEW

File: chkhealth.sh [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      exit 1
+      ;;
+    *)
+      POSITIONAL_ARGS+=("$1") # save positional arg
+      shift # past argument
+      ;;
+  esac
+done
+
+set -- "${POSITIONAL_ARGS[@]}" # restore positional parameters
```
Comment (gabrielmer) [inline chkhealth.sh:27]: I see that the one copied to `nwaku-compose` doesn't have this part.

Why is that? what's the difference between both?

Comment (NagyZoltanPeter) [reply chkhealth.sh:27]: Er, I don't get it. This one is in nwaku-compose. nwaku pr: https://github.com/waku-org/nwaku/pull/2624/files
is goiing to make it same.

Comment (gabrielmer) [reply chkhealth.sh:27]: Oops sorry my bad, it got hidden in the diff 😶 
Thanks :))

**Review by gabrielmer — APPROVED** [review]
Thank you!

**Pull request merged**